crypto/digestible/derive/src/lib.rs

```diff
@@ -40,6 +149,7 @@ fn try_digestible_struct(
     };
 
     // call is a Vec<TokenStream> (I think)
```

Comment:
Might be worth being explicit about the type, i.e. `let call: Vec<TokenStream> = ` instead of commenting about its type.

Reply:
CLion with Rust does this:

![Screenshot from 2020-09-02 13-42-44](https://user-images.githubusercontent.com/35878/92034810-9ef8d500-ed22-11ea-99bd-e063e4c4688b.png)